pages/1.12/deploying-services/private-docker-registry/index.md

```diff
@@ -235,8 +235,45 @@ Follow these steps to add your Docker registry credentials to the [DC/OS Enterpr
 
    The Docker image will now pull using the provided security credentials.
 
+<a name="docker-repo-certs"></a>
+
+# Configuring the Docker registry to use a custom certificate
+As an alternative to providing user credentials to authorize access to the private Docker registry, you can use a custom certificate to authorize access to the private Docker registry.
+
+To configure a custom certificate for accessing the private Docker registry and DC/OS UCR:
+
+1. Create or identify a custom certificate that you want to use as a trusted certificate for accessing the Docker registry.
+
+    You can use OpenSSL, DC/OS Enterprise CLI, or another program for generating public and private keys, certificate requests, and encrypted client and server certificates.
+
+1. Download or copy the certificate to the appropriate location.
+
+   For example, if you want to configure the DC/OS `ca.crt` certificate as a trusted certificate, you can download a copy of the `ca.crt` file to a local directory and set it as trusted using a command similar to the following:
```

Comment:
We're missing a few steps here.   The file needs to be copied to two places on each agent:

`/etc/docker/certs.d/<registry_name>:<registry_port>/ca.crt`
`/var/lib/dcos/pki/tls/certs/<something>.crt`

The `<registry_name>` and `<registry_port>`  are specific to the customer's installation.  

So, perhaps it makes sense to say, "For the following examples, we assume the local docker registry is referenced as registry.mycompany.com:5000".